src/bin.js

```diff
@@ -1,94 +1,99 @@
 #!/usr/bin/env node
 /* eslint no-console: 0, no-process-exit: 0 */
-"use strict";
+'use strict';
 
-var program = require("commander");
-var confirm = require("confirm-simple");
-var singleLineLog = require("single-line-log").stdout;
+const program = require('commander');
+const confirm = require('confirm-simple');
+const singleLineLog = require('single-line-log').stdout;
 
-var clean = require("./index");
-var meta = require("../package.json");
+const clean = require('.');
+const meta = require('../package.json');
 
 program
-  .version(meta.version)
-  .usage("token")
-  .option("--debug", "Output extended error details")
-  .option("--user <value>", "Only cleanup given user or organization")
-  .option("-y, --yes", "Do not ask for confirmation")
-  .parse(process.argv);
+	.version(meta.version)
+	.usage('token')
+	.option('--debug', 'Output extended error details')
+	.option('--user <value>', 'Only cleanup given user or organization')
+	.option('-y, --yes', 'Do not ask for confirmation')
+	.parse(process.argv);
 
-program.on("--help", function() {
-  console.log("  Please get the OAuth token:");
-  console.log("");
-  console.log("  Head to https://github.com/settings/tokens/new");
-  console.log("  Create a token with permissions public_repo, delete_repo");
-  console.log("  Pass that in the CLI, enjoy!");
-  console.log("");
+program.on('--help', () => {
```

Comment:
Here and in other similar places, is the `function(){}` syntax being replaced by `() => {}` automatically by xo or by you manually?

If automatically, I don’t mind much, just surprised to see. If manually by you, I don’t mind much, but I don’t see much benefit in that effort.

Reply:
XO does this automatically with eslint’s `prefer-arrow-callback` but
1. For some reason it’s being ignored when run with Prettier (it’s not a conflicting style) https://github.com/xojs/xo/issues/498 so I used Lebab to do the conversion until that bug is fixed 
2. In many cases it’s shorter and cleaner, e.g. 

``` js
a.map(function (b) {
  return b.data;
});
// versus 
a.map(b => b.data);

Reply:
Ah, `prefer-arrow-callback`, thanks for telling me. Let’s keep the xo default.

Of course they’re nice with short functions with implicit returns, but for longer functions, they do not save space, but add two more punctuation chars. ¯\\\_(ツ)\_/¯

Reply:
> but for longer functions, they do not save space

I don't think that every happens. The only difference is in declaration vs expression:

```js
function a () {}
const a = () => {}
```

for which I agree with you and hasn't been touched by this PR because I think there weren't any declarations.

Reply:
We’re (presumably) not saving bytes, we’re saving visual noise that costs attention span. In that regard, `list.map(item => item * 2)` is by far better (has less visual noise) than `list.map(function(item){ return item * 2; })`. The same is not the case for multiline functions. In the example below, in my opinion, the version with the keyword is less visually noisy due to the keyword being more distinct and atomic than the ASCII art arrow:

```js
addEventListener("click", function(event){
  // ...
});

addEventListener("click", (event) => {
  // ...
});
```

This becomes increasingly problematic in very nested expressions due to the likelihood of the arrow syntax being more similar to other js operators and syntaxes:

```js
foo <= bar ? quux / 3 : f(function(){
  // perform operations
});

foo <= bar ? quux / 3 : f(() => {
  // perform operations
});
```

This is all subjective, so I don’t mean to speak the final word, just explaining what I mean.

Reply:
I see what you mean. Many functions/callbacks will be gone in my promisification though, so it won't be much of a problem either way. ✌️